src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -7709,6 +7709,7 @@ public enum NormalizationForm
         [CLSCompliant(false)]
         public static bool TryCreate(uint value, out Rune result) { throw null; }
         public bool TryEncode(Span<char> destination, out int charsWritten) { throw null; }
+        public bool TryEncodeToUtf8Bytes(Span<byte> destination, out int bytesWritten) { throw null; }
```

Comment:
There are tests in `src\System.Runtime\tests\System\Text\RuneTests.netcoreapp.cs` that exercise this using reflection. They can be switched to not use reflection with this.